README.md

```diff
@@ -0,0 +1,76 @@
+# AI Palatal Rugoscopy
+
+Instance segmentation of palatal rugae patterns using Deep Learning.
+
+
+## Development environment
+By default, we use a [Docker](https://www.docker.com/) container as development environment. \
+Before building the image, please ensure you have the following resources available:
+
+- GPU with an updated driver version
+- nvidia-container-tookit (allows running GPU-accelerated containers)
+
+The script [build-container.sh](docker/build-container.sh) automatically builds the image and creates the container. To execute this script, run the following command in a bash terminal:
+
+```shell
+cd docker
+bash build_container.sh --data-path <DATA_PATH>
+```
+Where:
+```
+DATA_PATH is the path to a folder containing the dataset (images).
+```
+
+## Dataset Preprocessing
+There are a few scripts available to preprocess data. They can be found in the [standalone](src/standalone/) directory.
+
+- `dataset_split.py`: Splits a dataset into `train`, `test` and `validation` sets. Usage: 
+```shell
+python dataset_split.py --annotations-path <ANNOTATION_FILE> --output-path <OUTPUT_DIRECTORY> --split <TRAIN_SIZE> <VALIDATION_SIZE> [<TEST_SIZE>]
+```
```

Comment:
You could indent with 2 spaces if you want this code block to align with the text in the bullet.